Source/WebCore/platform/graphics/FloatPolygon.cpp

```diff
@@ -86,11 +86,10 @@ static unsigned findNextEdgeVertexIndex(const FloatPolygon& polygon, unsigned ve
 FloatPolygon::FloatPolygon(Vector<FloatPoint>&& vertices, WindRule fillRule)
     : m_vertices(WTFMove(vertices))
     , m_fillRule(fillRule)
+    , m_empty(m_vertices.size() < 3)
```

Comment:
Not sure this change makes things significantly better, although I suppose construction is usually better than assignment.